source/beamline/IDEAS/IDEASBeamline.cpp

```diff
@@ -27,6 +27,9 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "beamline/AMAdvancedControlDetectorEmulator.h"
 
+#include <QApplication>
```

Comment:
If we're going to be pedantic on the QApplication includes, there is one here too.